docs/posts/prepare.md

```diff
@@ -5,20 +5,46 @@ description: Preparing your data
 ---
 # Data format requirements
 
-If your data is in `h5ad` file (from the [`anndata`](https://anndata.readthedocs.io/en/latest/index.html) library) and meets the following requirements, you can go straight to `cellxgene launch`:  
+If your data is in `h5ad` file (from the [`anndata`](https://anndata.readthedocs.io/en/latest/index.html) library) and meets the following requirements, you can go straight to `cellxgene launch`:
 
-- Expression values (raw or normalized) in `anndata.X`  
+- Expression values (raw or normalized) in `anndata.X`
 - At least one embedding (e.g., tSNE, UMAP) in `anndata.obsm`, specified with the prefix `X_` (e.g., by default scanpy stores UMAP coordinates in `anndata.obsm['X_umap']`)
-- A unique identifier for every cell is available in an `anndata.obs` field (you can specify this with the `--obs-names` option)  
-- A unique identifier for every gene is available in an `anndata.var` field (you can specify which field to use with the `--var-names` option)  
+- A unique identifier for every cell is available in an `anndata.obs` field (you can specify this with the `--obs-names` option)
+- A unique identifier for every gene is available in an `anndata.var` field (you can specify which field to use with the `--var-names` option)
 
-#### What about R objects from seurat / bioconductor!?  
+#### What about R objects from seurat / bioconductor!?
 We hear you! We'd also love to be able to ingest these files directly. This isn't currently possible, but in the meantime, you can use [sceasy](https://bioconda.github.io/recipes/r-sceasy/README.html) ([docs](https://cellgeni.readthedocs.io/en/latest/visualisations.html)) to convert to `h5ad`. Seurat also has some [handy conversion tools](https://satijalab.org/seurat/v3.0/conversion_vignette.html) that you can try out.
 
-#### Can I use data hosted on the web somewhere?  
+#### Can I use data hosted on the web somewhere?
 Yes! You can launch from a URL instead of a filepath. The same data format requirements apply. Please see [here](launch) for more details.
 
-# Using `cellxgene prepare`  
+# Data format options
+
+#### Category colors
```

Comment:
You might want to add a note about disabling this behavior, as the app will _always_ use colors if present, which is going to cause a change in default behavior for existing datasets.